ci/indy-pool.dockerfile

```diff
@@ -26,7 +26,7 @@ RUN useradd -ms /bin/bash -u $uid indy
 
 ARG indy_plenum_ver=1.6.641
 ARG indy_anoncreds_ver=1.0.32
-ARG indy_node_ver=1.6.740
+ARG indy_node_ver=1.6.756
```

Comment:
Yes.  It worked for me.  I can update to .757 though.   What was the errors you saw?

Reply:
It can't work because of Indy-Node 1.6.756 depended on another version indy-plenum which also must be changed. 
Moreover, I found that 1.6.756 was published with the wrong dependency on distro and was fixed in this PR: https://github.com/hyperledger/indy-node/pull/1126/files

Reply:
here is output from jenkins ci
`[Ubuntu Test: build and test libindy] The following packages have unmet dependencies:
[Ubuntu Test: build and test libindy]  indy-node : Depends: indy-plenum (= 1.6.652) but 1.6.641 is to be installed
[Ubuntu Test: build and test libindy]              Depends: distro (= 1.3.0) but it is not installable
[Ubuntu Test: build and test libindy] [91mE: Unable to correct problems, you have held broken packages.`

Reply:
updated.